config_defaults.yml

```diff
@@ -1,29 +1,72 @@
 ---
 .travis.yml:
-  script: "\"bundle exec rake validate && bundle exec rake lint && bundle exec rake spec SPEC_OPTS='--format documentation'\""
+  supported_oss: &supported_oss
+    puppetlabs-acl:                      [windows]
+    puppetlabs-chocolatey:               [windows]
+    puppetlabs-dism:                     [windows]
+    puppetlabs-dsc:                      [windows]
+    puppetlabs-iis:                      [windows]
+    puppetlabs-powershell:               [windows]
+    puppetlabs-reboot:                   [windows]
+    puppetlabs-registry:                 [windows]
+    puppetlabs-sqlserver:                [windows]
+    puppetlabs-wsus_client:              [windows]
+    puppetlabs-inifile:                  [linux,windows]
+    puppetlabs-stdlib:                   [linux,windows]
```

Comment:
I would prefer the Gemfile to be smart enough to understand which platform it is running on, instead of hardcoding these here.